test/web-platform.js

```diff
@@ -54,3 +54,40 @@ describe("To-upstream tests", () => {
     it("Parsing: <" + expected.input + "> against <" + expected.base + ">", testURL(expected));
   }
 });
+
+describe("MDN examples", () => {
+  it("Checking all examples on MDN pass", () => {
+    const a = new URL("/", "https://developer.mozilla.org");
+    assert.strictEqual(a.href, "https://developer.mozilla.org/", "a");
```

Comment:
May I ask why? They correspond to the MDN examples and help determine which of the `assert` statements are actually failing. Without them, it would be difficult to see since many of the expected strings are identical.